### ISSUE TITLE
Create dedicated section on pipeline / art creation

### ISSUE THREAD

**Issue by rdb** [panda3d/panda3d-docs#26]
We have sections on programming, debugging, distribution, optimization, but we don't have a section dedicated to authoring content.  We have some documentation on the various exporters available, but it's tucked away in a Tools section, which currently looks like a bit of a pitiful collection of scraps, much of which outdated.

In order to make artists first-class citizens, one of the things I would propose is to create a section dedicated to explaining the Panda3D material and lighting models from a conceptual level (to the degree that artists need to understand it), pages on how to create new artwork, finding existing assets, conversion, tweaking parameters, best practices, etc., updated for modern tools like Blender 2.8.

I would then move the few pages under Tools to the more relevant section (eg. Interrogate should be under the section on writing C++ extensions, under Programming)

Does this seem like a good idea, or is it better to try to merge anything related to authoring content into the Programming with Panda3D section?

**Comment by Moguri** [comment]
+1 from me.

A section on authoring content and what artists need to be aware of when using Panda3d would be very helpful

**Comment by rdb** [comment]
Working on an Asset Pipeline section.

I am also going to delete the Milkshape3D and GMax pages in the 1.11 manual because they seem to be abandonware.

**Comment by rdb** [comment]
Closing this because I've made a start of this new section.  More content can be added later.  We can probably put Assimp and glTF stuff in dedicated pages, for example, with specific notes about limitations/caveats/configuration.  High-quality contributions would be welcome.

The stuff on DIRECT Tools I've merged into the Debugging section.  I deleted the Tools section altogether, but there are two dangling pages: Interrogate, and List of Executables.  Not quite sure what to do with these, hmm.  The latter could be part of the reference (if we need it at all) and the former could perhaps be moved either to a section in Programming on writing C++ extensions or outside the Panda3D manual altogether.
